src/Jh/DataImportMagento/Writer/ShipmentWriter.php

```diff
@@ -45,25 +45,45 @@ public function writeItem(array $item)
         if (!isset($item['order_id'])) {
             throw new WriterException('order_id must be set');
         }
-
         $order = clone $this->orderModel;
         $order->loadByIncrementId($item['order_id']);
-
         if (!$order->getId()) {
             throw new WriterException(sprintf('Order with ID: "%s" cannot be found', $item['order_id']));
         }
 
+        $customerEmail = $order->getCustomerEmail();
+
         try {
             $shipment = $order->prepareShipment();
             $shipment->register();
             $shipment->getOrder()->setData('is_in_process', true);
 
+            if (isset($customerEmail)) {
+                $shipment->sendEmail(
+                    true,
```

Comment:
This is very specific to the project you are working on, this cannot really go here, or you need to figure a way to abstract it.